misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/reporting/TextExplanatoryAssertionGroupFormatterSpec.kt

```diff
@@ -5,16 +5,18 @@ import ch.tutteli.atrium.assertions.ExplanatoryAssertionGroupType
 import ch.tutteli.atrium.domain.builders.ExpectImpl
 import ch.tutteli.atrium.reporting.AssertionFormatter
 import ch.tutteli.atrium.reporting.AssertionFormatterController
+import org.spekframework.spek2.Spek
 import kotlin.reflect.KClass
 
 //TODO #116 migrate spek1 to spek2 - move to specs-common
```

Comment:
don't forget to remove this TODO as well

Reply:
done